yarn.lock

```diff
@@ -7765,7 +7765,7 @@ matrix-react-test-utils@^0.2.3:
     "@babel/traverse" "^7.13.17"
     walk "^2.3.14"
 
-matrix-widget-api@^0.1.0-beta.16:
+matrix-widget-api@^0.1.0-beta.15, matrix-widget-api@^0.1.0-beta.16:
```

Comment:
This has the problem of now bundling two widget APIs which will break Jitsi calls at release. It needs to exactly match the react SDK.

Reply:
Looking into it.